app/assets/src/components/views/PipelineViz.jsx

```diff
@@ -125,7 +128,19 @@ class PipelineViz extends React.Component {
           intraEdgeData.push({ from: fromId, to: i });
         } else {
           // Connect beginning steps to input node
-          intraEdgeData.push({ from: START_NODE_ID, to: i });
+          intraEdgeData.push({
+            from: START_NODE_ID,
+            to: i,
+            ...(index == 0
+              ? {
+                  color: {
+                    color: backgroundColor,
```

Comment:
Is this background color for an edge? Why does it need background color?

Reply:
In the mocks the first stage doesn't have an entry edge to the first step, but in order to center the graph vertically I need to connect into the starting node, so the first edge to the first step of the first stage has a background color the same as the background to hide it. 

I'll add a comment in the code that explains this!

Reply:
Can you just make it 100% transparent?

Reply:
Done — didn't realize edges had a `color.opacity` field